adserver/staff/views.py

```diff
@@ -81,54 +81,75 @@ class PublisherPayoutView(StaffUserMixin, TemplateView):
     A view listing all the payouts that are due in the upcoming month.
 
     Has the following arguments:
-    * all: Show all publishers, not just folks with payouts due
+    * first: Show only payouts for the first time
+    * paid: Show only payouts that have already been paid
     * publisher: Filter to a specific publisher
-    * limit: Show only up to ``limit`` publishers, mostly for testing & debugging
     """
 
     template_name = "adserver/staff/publisher-payout-list.html"
-    # Two hours should be enough to run most payouts
-    CACHE_SECONDS = 7200
+    # Cache for 24 hours so we can finish payouts
+    CACHE_SECONDS = 3600 * 24
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        limit = int(self.request.GET.get("limit", 0))
-        all_publishers = self.request.GET.get("all")
         publisher_slug = self.request.GET.get("publisher")
+        first = self.request.GET.get("first", "")
+        paid = self.request.GET.get("paid", "")
```

Comment:
```suggestion
        paid = self.request.GET.get("paid", "") == "paid"
```

You might consider something like this to just make it a boolean.

Reply:
I want 3 states, which I guess could be True/False/None.